### ISSUE TITLE
Shape not implemented yet

### ISSUE THREAD

**Issue by Ontheway361** [MPolaris/onnx2tflite#28]
PyTorch code：
def forward(self, x):
        ## x.shape  = [B, T, F]
        B, _, _ = x.shape
        x = x.unsqueeze(1).transpose(2, 3) 
        x = self.layers(x)
        x = torch.mean(x, dim=2, keepdim=True)
        x = x.view((B, self.last_channel, -1))
        x = x.transpose(1, 2)
        return x

when convert onnx model to tflite, builder.py throw a KeyError, maybe we need to implement Shape op.

**Comment by MPolaris** [comment]
Hi Ontheway361, you can implement it by yourself at first, and welcome PR.
